Show "Submit Feedback" link before login.

lucyhe opened the pull request:
tested manually and ran grunt test.

![image](https://cloud.githubusercontent.com/assets/8723127/6835007/3ab4a1c0-d30e-11e4-8699-5eeeeeeaa41a.png)


<!-- Reviewable:start -->
[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/uproxy/uproxy/1160)
<!-- Reviewable:end -->

jpevarnek commented:
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy/1160)

Reviewed files:
- src/generic_ui/polymer/feedback.ts @ r1
- src/generic_ui/polymer/root.html @ r1
- src/generic_ui/polymer/splash.html @ r1
- src/generic_ui/polymer/splash.ts @ r1

---

Tiny questions about closing this.

---

<sup>**[src/generic_ui/polymer/feedback.ts, line 7 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-JlMNEfROYKGrBTLJqhj)** ([raw file](https://github.com/uproxy/uproxy/blob/886b4ce94bf6882a36b47a88050b7ddc83463f98/src/generic_ui/polymer/feedback.ts#L7)):</sup>
Should probably rename this

Also, in general, would it be possible to switch to doing something like `settingsOpen` as a boolean flag and just not changing the view for it (since we just overlay it anyways)?

With that you could also combine it with the below and just make sure you had saved `previousView` beforehand and always turn off settings when you are successful in submitting feedback (would also probably help when using this from other views)

---

<sup>**[src/generic_ui/polymer/feedback.ts, line 15 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-JlMNUyYfMvixa8BnrSd)** ([raw file](https://github.com/uproxy/uproxy/blob/886b4ce94bf6882a36b47a88050b7ddc83463f98/src/generic_ui/polymer/feedback.ts#L15)):</sup>
Is this `TODO` still valid?

---


<!-- Sent from Reviewable.io -->

lucyhe commented:
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy/1160)



---

PTAL, thank you!

---

<sup>**[src/generic_ui/polymer/feedback.ts, line 7 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-JlMNEfROYKGrBTLJqhj)** ([raw file](https://github.com/uproxy/uproxy/blob/886b4ce94bf6882a36b47a88050b7ddc83463f98/src/generic_ui/polymer/feedback.ts#L7)):</sup>
Renamed to close(), deleted FEEDBACK view so saving the previous view wont be needed either

---

<sup>**[src/generic_ui/polymer/feedback.ts, line 15 \[r1\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-JlMNUyYfMvixa8BnrSd)** ([raw file](https://github.com/uproxy/uproxy/blob/886b4ce94bf6882a36b47a88050b7ddc83463f98/src/generic_ui/polymer/feedback.ts#L15)):</sup>
Removing in https://github.com/uProxy/uproxy/pull/1173

---


<!-- Sent from Reviewable.io -->

jpevarnek commented:
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy/1160)

Reviewed files:
- bower.json @ r2
- src/generic_ui/polymer/feedback.html @ r2
- src/generic_ui/polymer/feedback.ts @ r2
- src/generic_ui/polymer/root.html @ r2
- src/generic_ui/polymer/root.ts @ r2
- src/generic_ui/polymer/settings.ts @ r2
- src/generic_ui/polymer/splash.html @ r2
- src/generic_ui/polymer/splash.ts @ r2
- src/uproxy.ts @ r2





---

<sup>**[src/generic_ui/polymer/root.ts, line 65 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-Jlg4q-9ANhitNhvOlRE)** ([raw file](https://github.com/uproxy/uproxy/blob/774827048717393b8d78c625ea2e14a2a03af753/src/generic_ui/polymer/root.ts#L65)):</sup>
Typo here (s/Openly/Opening)

---

<sup>**[src/generic_ui/polymer/settings.ts, line 51 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-Jlg56ypnulEt4e5Xk1f)** ([raw file](https://github.com/uproxy/uproxy/blob/774827048717393b8d78c625ea2e14a2a03af753/src/generic_ui/polymer/settings.ts#L51)):</sup>
Should use single quotes (https://docs.google.com/document/d/1ZV70oTNWO00bs0RcMwqr3FvKJXA6smYQ2kZ8Zr8YZnw/edit#heading=h.2l4tti1b6qwj)

---

<sup>**[src/generic_ui/polymer/splash.ts, line 28 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-Jlg6IlVBVW9z__uUvC-)** ([raw file](https://github.com/uproxy/uproxy/blob/774827048717393b8d78c625ea2e14a2a03af753/src/generic_ui/polymer/splash.ts#L28)):</sup>
Single quotes again

---


<!-- Sent from Reviewable.io -->

lucyhe commented:
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy/1160)







---

<sup>**[src/generic_ui/polymer/root.ts, line 65 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-Jlg4q-9ANhitNhvOlRE)** ([raw file](https://github.com/uproxy/uproxy/blob/774827048717393b8d78c625ea2e14a2a03af753/src/generic_ui/polymer/root.ts#L65)):</sup>
Done.

---

<sup>**[src/generic_ui/polymer/settings.ts, line 51 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-Jlg56ypnulEt4e5Xk1f)** ([raw file](https://github.com/uproxy/uproxy/blob/774827048717393b8d78c625ea2e14a2a03af753/src/generic_ui/polymer/settings.ts#L51)):</sup>
Done.

---

<sup>**[src/generic_ui/polymer/splash.ts, line 28 \[r2\]](https://reviewable.io:443/reviews/uproxy/uproxy/1160#-Jlg6IlVBVW9z__uUvC-)** ([raw file](https://github.com/uproxy/uproxy/blob/774827048717393b8d78c625ea2e14a2a03af753/src/generic_ui/polymer/splash.ts#L28)):</sup>
Done.

---


<!-- Sent from Reviewable.io -->

jpevarnek commented:
Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/uproxy/uproxy/1160)

Reviewed files:
- src/generic_ui/polymer/root.ts @ r3
- src/generic_ui/polymer/settings.ts @ r3
- src/generic_ui/polymer/splash.html @ r2
- src/generic_ui/polymer/splash.ts @ r3
- src/uproxy.ts @ r2

---

LGTM

---



<!-- Sent from Reviewable.io -->
